NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/TelemetryInitializers/ClientIpHeaderTelemetryInitializer.cs

```diff
@@ -125,30 +123,17 @@ protected override void OnInitializeTelemetry(HttpContext platformContext, Reque
             telemetry.Context.Location.Ip = requestTelemetry.Context.Location.Ip;
         }
 
-        private static string CutPort(string address)
+        private static bool ParseIPWithPort(string input, out string ipAddressString)
```

Comment:
nit: for this pattern of a method returning a bool with an out parameter, I would recommend naming it "TryParseIpWithPort".